NEWS.md

```diff
@@ -1,5 +1,9 @@
 # S7 0.0.0.9000
 
+## May-July 2023
+
+* `external_methods_register()` now registers S3 and S4 methods too (#306).
```

Comment:
I feel like the docs of this function could maybe use an update saying that this is supported